### PR TITLE
feat(msteams): Add notifications message builder

### PR DESCRIPTION
**Pull request by 97varun**
- Add a notifications message builder for issues builds cards with 4 fields for notifications.
  * Title
  * Attachment title
  * Message description
  * Footer

- Sample of how a card might look like,

image

- Make a slight modification in issue message builder for resue.
- Add tests for notification message builder

### CODE REVIEW

File: tests/sentry/integrations/msteams/test_message_builder.py [hunk 1]
```diff
@@ -360,3 +367,62 @@ def test_assigned_issue_message(self):
...
+        logo = footer["columns"][0]["items"][0]
+        assert "Image" == logo["type"]
+        assert "20px" == logo["height"]
+
+        footer_text = footer["columns"][1]["items"][0]
+        assert "Notification Footer" in footer_text["text"]
+        assert TextSize.SMALL == footer_text["size"]
+
+    def test_without_footer(self):
+        dummy_notifcation = DummyNotification(self.org)
```
Comment (ceorourke) [inline tests/sentry/integrations/msteams/test_message_builder.py:419]: nit: typo in notification

**Review by ceorourke — APPROVED** [review]

File: tests/sentry/integrations/msteams/test_message_builder.py (hunk 1 above)
Comment (97varun) [reply tests/sentry/integrations/msteams/test_message_builder.py]: Fixed!

**Review by scefali — COMMENTED** [review]
@97varun your PR has diffs from other commits, can you fix? I'll review then.

**Comment by 97varun** [comment]
> @97varun your PR has diffs from other commits, can you fix? I'll review then.

Done.

**Review by ceorourke — APPROVED** [review]

**Pull request merged**